tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -861,6 +861,46 @@ def _destroy_resource(self):
           handle, ignore_lookup_error=True)
 
 
+def print_row(fields, positions, print_fn):
```

Comment:
This should be a private method, _print_row

Reply:
Fixed